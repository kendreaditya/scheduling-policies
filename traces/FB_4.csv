1,1,1
2,2,2
3,3,3
4,4,4
5,90,10
6,100,1000
7,100,890
8,900,1
9,900,2
10,900,3
11,2000,3
12,2001,2
13,2002,1
14,2003,0
15,2003,1
16,2003,0
17,2004,0
18,2005,0
19,2005,3
20,2005,0
21,2006,0
22,2007,0
23,2008,0
24,2009,0
25,2010,0
26,2011,0
