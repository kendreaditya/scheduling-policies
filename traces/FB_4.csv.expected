1,2
2,8
3,10
4,11
5,100
6,1996
7,1886
8,903
9,905
10,906
11,2010
12,2008
13,2003
14,2003
15,2004
16,2003
17,2004
18,2005
19,2010
20,2005
21,2006
22,2007
23,2008
24,2009
25,2010
26,2011
